src/test/regress/create_schedule

```diff
@@ -3,3 +3,4 @@ test: prepared_statements_create_load ch_benchmarks_create_load
 test: dropped_columns_create_load distributed_planning_create_load
 test: local_dist_join_load
 test: partitioned_indexes_create
+test: connectivity_checks
```

Comment:
if we put this also to `sql_schedule`, it'd also run from the workers, which is a good coverage. 

If there are non-mx workers, this test might fail. But, as far as I remember all the tests run with MX enabled. So, can you also try adding there? If anything fails, I think running `citus_check_cluster_node_health` from the workers on one of the mx tests would be good sanity check

Reply:
There are many inaccessible worker nodes with different cluster names in MX tests.

```
SELECT * FROM pg_dist_node;
 nodeid | groupid | nodename  | nodeport | noderack | hasmetadata | isactive |  noderole   |                           nodecluster                           | metadatasynced | shouldhaveshards
---------------------------------------------------------------------
     16 |      14 | localhost |    57637 | default  | t           | t        | primary     | default                                                         | t              | t
     25 |      14 | localhost |     8888 | default  | f           | t        | secondary   | olap                                                            | f              | t
     30 |      18 | localhost |    57638 | default  | t           | t        | primary     | default                                                         | t              | t
     26 |      14 | localhost |     8887 | default  | f           | t        | secondary   | thisisasixtyfourcharacterstringrepeatedfourtimestomake256chars. | f              | t
     27 |      14 | localhost |     9995 | default  | f           | t        | secondary   | default                                                         | f              | t
     28 |      16 | localhost |     9994 | default  | f           | t        | secondary   | default                                                         | f              | t
     29 |      14 | localhost |     9992 | default  | f           | t        | secondary   | second-cluster                                                  | f              | t
     19 |      14 | localhost |     9998 | default  | f           | t        | secondary   | default                                                         | f              | t
     20 |      14 | localhost |     9997 | default  | f           | t        | unavailable | default                                                         | f              | t
     21 |      16 | localhost |     9996 | default  | f           | f        | secondary   | default                                                         | f              | t
(10 rows)
```

I ended up comparing the distinct counts of connectivity check results in the coordinator and the mx worker node.

```
SELECT result, count(*) FROM citus_check_cluster_node_health() GROUP BY result ORDER BY result;
 result | count
---------------------------------------------------------------------
 f      |    16
 t      |     4
        |    80
(3 rows)
```